quickstarts/gradle/webapp/build.gradle

```diff
@@ -26,3 +26,15 @@ compileJava {
 compileTestJava {
     options.encoding = 'UTF-8'
 }
+
+/*
+kubernetes {
```

Comment:
I was wondering if we could enable this block based on some property:
```groovy
if (project.hasProperty('enableJetty')) {
kubernetes {
    generator {
        config {
            'webapp' {
                server = "jetty"
            }
        }
    }
}
}
```

if we do this, the user won't need to go to `build.gradle` and comment/uncomment to switch between tomcat and jetty. Default behavior would be with tomcat and `gradle k8sBuild -PenableJetty` would enable jetty.

Reply:
I understand, but I would like to force the user to go and read/modify the build.gradle file if he which to modify it and try jetty (or wildfly) in a learning purpose. Does it make sense?

Reply:
sure, I'm totally fine with this.